### PR TITLE
Mehrdad.karami enhancement

### PR DESCRIPTION
**Pull request by metao1**
- Some cosmetic code cleanup
- Some typo fixed both in doc and method names

### CODE REVIEW

File: spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java [hunk 1]
```diff
@@ -1688,7 +1686,7 @@ private synchronized void captureOffsets(ConsumerRecords<K, V> records) {
 			}
 		}
 
-		private void checkRebalanceCommits() {
+		private void checkRebalancedCommits() {
```
Comment (garyrussell) [inline spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java:1689]: I prefer the existing name here.

File: spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java [hunk 2]
```diff
@@ -2331,19 +2332,17 @@ private void commitOffsetsIfNeeded(final ConsumerRecords<K, V> records) {
...
 		private void batchInterceptAfter(ConsumerRecords<K, V> records, @Nullable Exception exception) {
-			if (this.commonBatchInterceptor != null) {
-				try {
-					if (exception == null) {
-						this.commonBatchInterceptor.success(records, this.consumer);
-					}
-					else {
-						this.commonBatchInterceptor.failure(records, exception, this.consumer);
-					}
+			try {
```
Comment (garyrussell) [inline spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java:2335]: This is incorrect; the interceptor can be null; it probably should be marked `@Nullable`.

**Review by garyrussell — CHANGES_REQUESTED** [review]
Thanks.

I will look at this some more tomorrow, but a couple of comments so far.

**Review by artembilan — CHANGES_REQUESTED** [review]
Thanks for contribution but why did you name the PR after yourself? 
Why just don't provide the the head line from commit message which GH suggests by default?
Also see this article for best practice around commit messages: https://cbea.ms/git-commit/

Plus: did you run your fix locally - `gradlew clean check`?
Looks like there is some unreachable method still in use:
```
 /home/runner/work/spring-kafka/spring-kafka/spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java:3475: error: cannot find symbol
					checkRebalanceCommits();
```

**Comment by garyrussell** [comment]
That reference to `checkRebalanceCommits()` was in new code merged Friday.

**Comment by garyrussell** [comment]
That reference to `checkRebalanceCommits()` was in new code from a PR submitted Friday and merged today.

**Pull request closed without merging**